inji-verify-sdk/src/components/openid4vp-verification/OpenID4VPVerification.types.ts

```diff
@@ -159,4 +159,6 @@ export type OpenID4VPVerificationProps = ExclusivePresentationDefinition &
      * This is a required field to ensure proper error handling.
      */
     onError: (error: Error) => void;
+
+    errorMessageTemplate?: string;
```

Comment:
do we need to pass  errorMessageTemplate to sdk ? aren't we complicating sdk by adding all this additional fields ?